packages/modules/index/src/models/index-data.ts

```diff
@@ -0,0 +1,45 @@
+import {
```

Comment:
question: Are we missing something in the DML to use this here?

Reply:
The gin index perhaps?

Reply:
The GIN index and an autoincrement column

Reply:
yes this was missing but we tried DML first ahah